src/GafferSceneUI/ShaderView.cpp

```diff
@@ -391,34 +388,58 @@ void ShaderView::updateScene()
 		SceneCreators::const_iterator it = sc.find( prefixAndName );
 		if( it == sc.end() )
 		{
-			it = sc.find( PrefixAndName( prefixAndName.first, "Default" ) );
+			IECore::msg(
+				IECore::Msg::Error, "ShaderView",
+				fmt::format( "SceneCreator \"{}\" not registered", prefixAndName.second )
+			);
+			m_scene = nullptr;
```

Comment:
What's up with removing the fallback to "Default" here? Quite possibly reasonable, but I don't see any mention of this, and it kinda seems like it would be a change in behaviour? Though maybe behaviour that isn't used?

Reply:
In practice the behaviour isn't used, because we always register a "Default" in the UI, and we only let users choose scenes which exist. But this change is consistent with the new handling of SceneCreators that fail. If something doesn't exist, we show a black image, instead of falling back and giving the false impression that it's working.

On balance, it seemed better to not treat this as a breaking change and therefore to hold it for 1.5. The crash we were getting is real, due to bugs in a certain custom SceneCreator. Whereas I think the impacts of the fallback removal are purely theoretical.